config/routes.rb

```diff
@@ -18,6 +18,9 @@
   get 'reports/preview/:id', to: 'reports#preview', as: 'preview_report'
 
   get 'chatfuel/:topic', to: 'chatfuel#show'
+  # route for chatfuel questions and answers
+  get 'chatfuel/:topic/qa/:id', to: 'chatfuel#showQuestionAndAnswer', as: 'chatfuel_qa'
```

Comment:
In ruby land it is convention to use `snake_case` for methods and `CamelCase` for constants, e.g. class names. Constants require a leading upcase letter.